Module.yaml

```diff
@@ -57,6 +57,8 @@ envs:                                   # Array of environment variables that wi
      - 1
      - 1.5
      - 2
+ envDevices:
+      - PORT
```

Comment:
PORT is not a device.
Put the key on the env that has the device path

Reply:
Do you mean I have to put '/dev/ttyUSB0'?